content/en/docs/contributing/release-process.md

```diff
@@ -210,8 +240,8 @@ The process for cutting a minor release is as follows:
         export RELEASE_VERSION="1.3.0-alpha.0"
         ```
 
-   2. Generate `release-note.md` at the root of your cert-manager repo folder
-      with the following command:
+    2. Generate `release-note.md` at the root of your cert-manager repo folder
+       with the following command:
```

Comment:
Note when sanity checking the release notes - `START_REV` and `END_REV` are inclusive, so check you haven't duplicated the final release note from the previous release.

Reply:
Thanks! Done in 31c3690